ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -35,7 +35,7 @@ tests:
     test:
     - chain: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts
-- as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
+- as: perfscale-rosa-multiaz-ci-tests-24nodes-periodic
```

Comment:
Same here. Can name reflect that these are control plane tests instead of `ci-tests`

Reply:
Few thoughts here:
1) I am keeping this job name consistent with the workload in this case `openshift-qe-ocp-qe-perfscale-ci-tests`
2) Even thought I do not mind changing the job name to `control-plane-test`, it should also be reflected across other places in `ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/`

Reply:
@jtaleric ^^

Reply:
This change is blocked on
https://github.com/openshift/release/pull/42788

Reply:
The said change is merged.